cpu/Makefile.include.cortexm_common

```diff
@@ -15,6 +15,24 @@ export CFLAGS_OPT  ?= -Os
 export CFLAGS += $(CFLAGS_CPU) $(CFLAGS_LINK) $(CFLAGS_DBG) $(CFLAGS_OPT)
 
 export ASFLAGS += $(CFLAGS_CPU) $(CFLAGS_DBG)
+
+# If the FW_SLOTS flag is set, we will generate a linker script at compile-time
+# reflecting the FW_IMAGE_OFFSET and FW_IMAGE_LENGTH values defined in the
+# target project's Makefile.
+ifeq ($(FW_SLOTS),1)
+ifeq ($(FW_SLOT),1)
```

Comment:
The idea actually is to limit to 2. The next step is to try to get always slot 1 as the bootable slot, download images to slot 2 and then swap both images. The complexity of having several slots is that you need to know before compiling where are you putting them (e.g. to know where they are the current images to don't overwrite them).